crates/solver/src/arguments.rs

```diff
@@ -328,8 +328,28 @@ pub struct Arguments {
     pub s3_upload: S3UploadArguments,
 
     /// Enable ranking of settlements by score
+    /// /// CIP20 TODO - remove after transition period is over
     #[clap(long, env, default_value = "false")]
     pub enable_auction_rewards: bool,
+
+    /// The time between two mined blocks in seconds
+    #[clap(
+        long,
+        env,
+        default_value = "13",
+        value_parser = shared::arguments::duration_from_seconds,
+    )]
+    pub block_time: Duration, // TODO release note: set for each network in configuration
+
+    /// Additional time to wait for a transaction to appear onchain before
+    /// considering the solution invalid and setting the reward to 0.
```

Comment:
We need it for cases when our `current_block()` calls are not reliable, meaning the call maybe hits a delayed node. This would cause `block_deadline` to be sooner than expected, and transactions being mined at the end of real submission deadline period (but inside) to miss the reward.
Since I needed some additional value, I thought I better put it under configuration.

Reply:
Once we move this to autopilot it will make more sense I guess.

Reply:
Autopilot already has a configuration value for this because it was needed for colocation.

Reply:
@vkgnosis moved to `shared` and reused.